Dockerfile

```diff
@@ -8,6 +8,10 @@ EOF
 
 WORKDIR /app
 
+RUN GRPC_HEALTH_PROBE_VERSION=v0.4.23 \
+    && curl -Lfso /bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-${TARGETARCH} \
+    && chmod +x /bin/grpc_health_probe
+
```

Comment:
moved this up so we don't need curl in the container below

Reply:
Fair enough, but considering the size of `curl` I've often found it handy when debugging networking problems in prod. I'm not sure I'd want to give that up.

We will also be able to fully drop this after our next K8S upgrade, they built this into core K8S as of version 1.27.

Reply:
This is no longer a thing